python/ml4ir/base/model/cyclic_learning_rate.py

```diff
@@ -0,0 +1,281 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Cyclical Learning Rate Schedule policies for TensorFlow."""
```

Comment:
@jakemannix We are using source code from tf_addons directly here as tf_addons is not supported on tf.2.0.3 yet. We will remove this when we upgrade to tf2.3. Hope that this is fine with you. It seems to be working out of the box and gives us new tools to optimize.